eta-meeting-organizer-frontend/src/app/building/components/building.component.ts

```diff
@@ -1,45 +1,48 @@
 import { Component } from '@angular/core';
-import { FormControl, FormGroup } from '@angular/forms';
+import { Observable } from 'rxjs';
+import { ApiCommunicationService } from '~/app/shared/services/api-communication.service';
+import { Building } from '~/app/models/building.model';
 
 @Component({
   selector: 'app-building',
```

Comment:
ezt a selectort cseréljük le meg a komponens nevét is valami olyanra ami elmondja pontosan mi történik itt

Reply:
javítottam app-building-list -re